doc/QUICKLOOK/qa-sky-CAMERA-EXPID.rst

```diff
@@ -0,0 +1,73 @@
+========================
+qa-sky-CAMERA-EXPID.yaml
+========================
+
+:Summary: This QA for QuickLook first includes the calculation of 
+        signal-to-noise in the spectrum after sky subtraction. 
+:Naming Convention: ``qa-{ARM}{SPECTROGRAPH}-{EXPID}.yaml``, where 
+        {ARM} is the 1-char arm name (r,b,z), {SPECTROGRAPH} indexes 
+        CCDs 0-9 on that arm, and {EXPID} is the 8-digit exposure ID.  
+        Together, {ARM}{SPECTROGRAPH} specify a {CAMERA}.
+:Regex: ``qa-sky-[brz][0-9]-[0-9]{8}\.yaml``
+:File Type:  YAML
+
+
+Inputs
+======
+
+Written by qa_quicklook.py, with Calculate_SNR using:
+
+- frame
+- skymodel
+
+Contents
+========
+
+========== ================ ===========================
+KEYNAME    Type             Contents
+========== ================ ===========================
+SNR        Py dictionary    Spectral signal-to-noise
+========== ================ ===========================
+
+
+
+Dictionary Contents
+===================
+
+KEYNAME = SNR
+
+Spectral signal-to-noise calculations per fiber and per CCD amp.
+
+Keyword Description
+~~~~~~~~~~~~~~~~~~~
+
+================ ============= ========== ============
+KEY              Example Value Type       Comment
+================ ============= ========== ============
+ARM              r             char       Spectrograph arm b,r,z
+SPECTROGRAPH     0             int  	  Camera index 0..9
+EXPID            00000002      int  	  exposure ID
+QANAME		 SNR           string     name of QA algorithm
+TOTAL_SNR        44.484        float[500] summed S/N per fiber 
+MEDIAN_SNR       1.3233        float[500] median S/N per fiber
+TOT_AMP_SNR	 382.74	       float[4]   summed S/N averaged over each amplifier
+MEDIAN_AMP_SNR	 1.7905	       float[4]   median S/N averaged over each amplifier
+================ ============= ========== ============
+
+Example YAML Output (10 spectra)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+

```

Comment:
This will look weird because it will not be in fixed width format.  See http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#literal-blocks for creating literal blocks (like `<pre>` in HTML).